workflows/cmp-cv/swift/workflow.sh

```diff
@@ -17,7 +17,7 @@ usage()
   echo "CMP-CV: usage: workflow.sh SITE EXPID CFG_SYS PLAN"
 }
 
-if (( ${#} != 4 ))
```

Comment:
This will break other workflows that supply 4 arguments.

Reply:
Since this workflow is only applicable for cmp-cv, adding this workflow.sh will not affect the other workflows, right?